### PR TITLE
Enable end-to-end tests on CI

### PR DESCRIPTION
**Pull request by klochowicz**
First run will probably take ages because of lack of cache.
Will closely monitor and optimise onwards.

I'm trying to remove the flutter dependency.

### CODE REVIEW

File: justfile
Comment (luckysori) [inline justfile:1]: 🔧 Can't we split it so that we build separately and only start waiting when we run it?

**Review by luckysori — APPROVED** [review]
Nice, it seems to work already!

File: justfile
Comment (klochowicz) [reply justfile:1]: I ended up doing it anyway :) but 10s was really arbitrary, so I thought we might as well include it.

**Pull request merged**